code/obj/grille.dm

```diff
@@ -519,6 +522,17 @@
 
 		return src.electrocute(user, prb, net, ignore_gloves)
 
+	proc/lightningrod(lpower)
+		if (!anchored)
+			return FALSE
+		var/net = get_connection()
+		if (!powernets[net])
+			return FALSE
+		if (src.material.hasProperty("electrical")) // fun fact: you get -1 if this isnt here
+			powernets[net].newavail += lpower/100 * (100 - src.material.getProperty("electrical"))
+		else
+			powernets[net].newavail += lpower / 100 * 40 // default to 60
```

Comment:
where is 60

Reply:
changed it to (100 - 60), the "electrical" property itself has default_value set to 60